redis/README.rst

```diff
@@ -8,7 +8,7 @@ Redis instances on the Google Cloud Platform.
 - `Product Documentation`_
 
 .. _Alpha: https://github.com/GoogleCloudPlatform/google-cloud-python/blob/master/README.rst
-.. _Google Cloud Memorystore for Redis API: https://cloud.google.com/redis
+.. _Google Cloud Memorystore for Redis API: https://cloud.google.com/memorystore
```

Comment:
Hmm, both the old and the new links 404 for me. :(  The railing slash needs to be there (BOOO, Hisssssss!)